syntax = "proto3";

message callerRecord {
    string FullName = 1;
    string PhoneNumber = 2;
}

message ackRecord {
    string message  = 1;
}
service recorder {
    rpc ingestRecord (callerRecord) returns (ackRecord);
}
